php/php.phing/src/org/netbeans/modules/php/phing/exec/PhingExecutable.java

```diff
@@ -163,10 +161,7 @@ private PhpExecutable getExecutable(String title) {
     }
 
     private List<String> getRunParams(String[] args) {
-        List<String> params = new ArrayList<>(args.length + 2);
-        params.add(LOGGER_PARAM);
-        params.add(LOGGER_COLOR_VALUE);
-        params.addAll(Arrays.asList(args));
+        List<String> params = Arrays.asList(args);
```

Comment:
Sorry, still different semantics - before you got a `java.util.ArrayList`, after the change it will be `java.util.Arrays.ArrayList`. The latter is modifiable, but has fixed length, which is not true for the first.

Reply:
Hopefully it is correct this time.